src/findfonts.jl

```diff
@@ -118,56 +142,53 @@ function try_load(fpath)
         return nothing
     end
 end
+try_load(::Nothing) = nothing
 
 fontname(ft::FTFont) = "$(family_name(ft)) $(style_name(ft))"
 
+const FONT_CACHE = Dict{String, Tuple{Float64, Union{String, Nothing}}}()
+
 function findfont(
         searchstring::String;
         italic::Bool=false, # this is unused in the new implementation
         bold::Bool=false, # and this as well
         additional_fonts::String=""
     )
     font_folders = copy(fontpaths())
+    isempty(additional_fonts) || pushfirst!(font_folders, additional_fonts)
 
+    font_folders = copy(fontpaths())
     isempty(additional_fonts) || pushfirst!(font_folders, additional_fonts)
+    filter!(isdir, font_folders)
```

Comment:
maybe this should error instead when any font folder is not a dir? or is this expected

Reply:
It's an edge-case, but it's theoretically possible for a font-dir to disappear after `fontpaths()` is established (e.g. if a Linux user did `mv ~/.fonts ~/.local/fonts`), and I think it would be better not to error in such a situation.

Reply:
ok sounds reasonable